feat(connect-chaos-infrastructures): Add connect chaos infrastructures docs (#1)
chaosnative/developer-hub-cn: docs/chaos-engineering/User Guides/connect-chaos-infrastructures.md

## Create Environment
To add a chaos infrastructure for your target environment, first we need to create a new environment as part of which the chaos infrastructures will exist. Head to the **Environments** page, which lists all the environments added as part of your project. Choose **New Environment**.

![Chaos Environments](./static/connect-chaos-infrastructures/chaos-environments.png)

In the **Create a new environment** modal, provide a name for the environment, optionally you can also add a description and tags. Then select an environment type i.e. **Production** or **Non-Production**. Then select **Create**. This will instantly create a new environment.

> ![New Environment](./static/connect-chaos-infrastructures/new-environment.png)

Reviewer: This screen has been updated in the [stress](https://stress.harness.io/) can you please update it from there

## Add Chaos Infrastructure
Now we can add a chaos infrastructure to our environment. Select **New Chaos Infrastructure**.

> You can either setup a chaos infrastructure on existing infrastructures that uses a Harness cluster connector i.e. Harness Kubernetes delegate or install the chaos infrastructure in a new infrastructure.

Reviewer: ```suggestion
You can either setup a chaos infrastructure on existing infrastructures that uses a Harness cluster connector i.e. Harness Kubernetes connector or install the chaos infrastructure in a new infrastructure.
```
Today we list only harness connectors for the brownfield method, wherein the harness delegate (which is being pointed at by the actual harness connector) is already configured...
directly deploying using a harness Kubernetes delegate is a future scope item

> To use an existing Harness Kubernetes delegate to install your chaos infrastructure, select **On Existing Infrastructures** in the modal. You can use any delegate under the Project, Organization, and Account scope.

Reviewer: ```suggestion
To use an existing Harness Kubernetes connector to install your chaos infrastructure, select **On Existing Infrastructures** in the modal. You can use any delegate under the Project, Organization, and Account scope.
```

![Chaos Infrastructure in Existing Infra](./static/connect-chaos-infrastructures/chaos-infrastructure-in-existing-infra.png)

:::info
1. A Harness Kubernetes delegate with cluster-wide read/write access can setup both cluster-scoped and namespaced-scoped chaos infrastructures.
2. A Harness Kubernetes delegate with cluster-wide read access **can't** be used to setup a chaos infrastructures.

> 3. A Harness Kubernetes delegate with specific namespace access can only be used to setup a namespace-scoped chaos infrastructure in that specific Kubernetes namespace.

Reviewer: I think here we may be referring to harness Kubernetes connectors with their specific harness Kubernetes Delegates configured to the particular settings we are describing here, ignore if this was not the intention

:::

To install chaos infrastructure on a new infrastructure, select **On New Infrastructures**. Then, select **Continue**.
Provide a name for the infrastructure. Optionally, you can also provide a description and tags. Then, select **Next**. 

> Under the installation mode, choose either **Cluster Wide** or **Namespace Mode** for installing the chaos infrastructure. By default the installation will take place in the `litmus` namespace and uses `litmus` service account, which can be configured under the K8s Cluster Details. Optionally, you can also specify node selectors and K8s tolerations for chaos infrastructure deployment.

Reviewer: For the brownfield method (using harness Kubernetes connector) if a user has a pre-existing cluster wide installation of the chaos infrastructure on a particular connector, they will no longer be presented with the cluster wide option again (for that specific connector), as cluster scope installation of the chaos infrastructure overrides the existing cluster resources, we may want to add this note here

> ![Configure Chaos Infrastructure](./static/connect-chaos-infrastructures/configure-chaos-infrastructure.png)

Reviewer: This screen (color related changes) has been updated in the [stress](https://stress.harness.io/) can you please update it from there

Then, select **Next**. In the final step, if you're deploying to an existing infrastructure with Harness Kubernetes Delegate, then Harness will install the chaos infrastructure on your behalf. Otherwise, if you're setting up the chaos infrastructure on a new infrastructure, then you will need to execute the given commands and/or download and apply the installation manifest.

![k8s Setup Infrastructure](./static/connect-chaos-infrastructures/k8s-setup-infrastructure.png)

Finally, select **Done**.

## Validating Chaos Infrastructure Installation
Upon applying the manifest, it will take a while to setup all the chaos infrastructure resources. After a while, the chaos infrastructure's connection state should reflect as `CONNECTED`.

> If the chaos infrastructure remains in a `NOT CONNECTED` state, follow the suggestions on the **Troubleshooting** tab to resolve it.

Reviewer: ```suggestion
If the chaos infrastructure remains in a `INACTIVE` / `PENDING` state, follow the suggestions on the **Troubleshooting** tab to resolve it.
```

> ![Infrastructure State](./static/connect-chaos-infrastructures/infrastructure-state.png)

Reviewer: This screen has been updated in the [stress](https://stress.harness.io/) can you please update it from there
